DVSRegister/Controllers/PreRegistrationController.cs

```diff
@@ -75,13 +78,33 @@ public IActionResult SaveApplicationSponsorSelection(SummaryViewModel viewModel)
         [HttpGet]
         public IActionResult Sponsor()
         {
+            // Retrieve form validation and first visit info from Session
+            ViewBag.IsFirstVisit = HttpContext?.Session.GetString("IsFirstVisit");
+            ViewBag.IsFullNameValid = HttpContext?.Session.GetString("IsFullNameValid");
+            ViewBag.IsEmailValid = HttpContext?.Session.GetString("IsEmailValid");
```

Comment:
Is it possible to make as private method? private void RetrieveValidationVariables ?

Reply:
Done